src/math/rng/probability.rs

```diff
@@ -247,7 +247,7 @@ impl Probability {
                 ref offsets,
                 ref areas,
                 ref cdf,
-                ref xs,
+                xs: _,
```

Comment:
Not sure about the correct answer to this, but I wonder if `ref xs: _` might be more optimal than `xs: _`?

Even though the value `xs` isn't used does the compiler still copy it (rather than just take it by `ref`)?

My feeling is that it doesn't matter because the compiler will drop it if it's unused anyway

Reply:
`ref xs: _` throws syntax errors, so I don't think this is valid syntax. I think the compiler is just dropping the arg, hence no need to worry about `ref`.